packages/e2e-tests/helpers/utils.ts

```diff
@@ -179,3 +179,53 @@ export async function resetTestUser(email: string) {
     },
   });
 }
+
+export async function cloneTestRecording(recordingId: string): Promise<string> {
+  const variables = { recordingId, secret: process.env.AUTOMATED_TEST_SECRET };
+
+  var startTime = performance.now();
+  const clonedRecording = await axios({
+    url: config.graphqlUrl,
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${process.env.AUTHENTICATED_TESTS_WORKSPACE_API_KEY}`,
+    },
+    data: {
+      query: `
+        mutation cloneTestRecording($recordingId: UUID!, $secret: String!) {
+          cloneTestRecording(input: { recordingId: $recordingId, secret: $secret }) {
+            recordingId
+          }
+        }
+      `,
+      variables,
+    },
+  });
+  var endTime = performance.now();
+  const timeInSecs = (endTime - startTime) / 1000;
+  console.log(`Cloning took ${timeInSecs} seconds.`);
```

Comment:
Remove before landing?

```suggestion
```

Reply:
There isn't any observability into tests right now. And in future, if we ever had a big example recording it'd be hard to point if test itself is flow or machine it's running on or the clone.

How about we log if time is greater than 10 sec?

Reply:
Hm, gotcha. That's fair. Maybe `console.warn` then? 😄